File: 闲来无事/Effective Objective-C 2.0/第2章以前/Number_Five/Number_Five/ViewController.m
//
//  ViewController.m
//  Number_Five
//
//  Created by huangaengoln on 15/12/16.
//  Copyright © 2015年 huangaengoln. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    EOCConnectionState _currentState;
    switch (_currentState) {
        case EOCConnectionStateDisconnected:
            // Handle disconnected state
            break;
        case EOCConnectionStateConnecting:
            // Handle disconnected state
            break;
        case EOCConnectionStateConnected:
            // Handle disconnected state
            break;
    }
    
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: 闲来无事/SwiftDemo/SwiftDemo/ViewController.swift
//
//  ViewController.swift
//  SwiftDemo
//
//  Created by huangaengoln on 16/2/27.
//  Copyright © 2016年 huangaengoln. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    
    var tableView:UITableView?
    var controllersArray:NSMutableArray?
    var titlesArray:NSMutableArray?

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        
        self.title="SwiftDemo"
        
        self.view.backgroundColor=UIColor.whiteColor()
        
        
        
        
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

